lib/mainloop-call.c

```diff
@@ -135,17 +150,26 @@ main_loop_call_handler(gpointer user_data)
 void
 main_loop_call_thread_init(void)
 {
-  g_cond_init(&call_info.cond);
-  g_mutex_init(&call_info.lock);
+  call_info = g_malloc(sizeof (MainLoopTaskCallSite));
```

Comment:
Let's 0 init all the fields:

```suggestion
  call_info = g_new0(MainLoopTaskCallSite, 1);
```